packages/ui/stories/molecules/RadioSelect.stories.tsx

```diff
@@ -29,6 +29,8 @@ storiesOf('ui/molecules/RadioSelect', module).add(
           {id: '0', title: 'Option 1'},
           {id: '1', title: 'Option 2'},
         ]}
+        direction="row"
+        onChange={() => console.log('Item has changed.')}
```

Comment:
Why ? direction default value is defined in args and onChange is not useful in storybook